### PR TITLE
Add domains import resources

### PR DESCRIPTION
**Pull request by szok**

### CODE REVIEW

File: src/ralph/data_importer/widgets.py [hunk 1]
```diff
@@ -182,11 +182,14 @@ def clean(self, value):
...
         try:
-            value = value.split("|")  # service, enviroment
-            value = ServiceEnvironment.objects.get(
-                service__name=value[0],
-                environment__name=value[1]
-            )
+            if value.isdigit():
+                value = ServiceEnvironment.objects.get(pk=value)
+            else:
+                service, enviroment = value.split("|")
```
Comment (mkurek) [inline src/ralph/data_importer/widgets.py:188]: we should catch `ValueError` here as well

File: src/ralph/data_importer/widgets.py [hunk 2]
```diff
@@ -182,12 +182,15 @@ def clean(self, value):
...
+                value = ServiceEnvironment.objects.get(
+                    service__name=service,
+                    environment__name=enviroment
+                )
+        except (ValueError, ServiceEnvironment.DoesNotExist):
             value = None
```
Comment (mkurek) [inline src/ralph/data_importer/widgets.py:194]: shouldn't we raise exception here? Is this row going to be imported - in my opinion it shouldn't

**Comment by mkurek** [comment]
:ok:

File: src/ralph/data_importer/widgets.py (hunk 2 above)
Comment (ghost) [inline src/ralph/data_importer/widgets.py:194]: :+1:

**Comment by ghost** [comment]
:ok:

**Pull request merged**